Tests/SwiftCompilerPluginTest/LRUCacheTests.swift

```diff
@@ -11,6 +11,7 @@
 //===----------------------------------------------------------------------===//
 
 @_spi(Testing) import SwiftCompilerPluginMessageHandling
+import SwiftSyntax
```

Comment:
If `LRUCache` is in SwiftSyntax`, I think we should also move the tests to SwiftSyntax.